From aeed12accf9a8fa582343c35e846434401fc0a7b Mon Sep 17 00:00:00 2001
From: Shubhang Rathore <shubhang92@gmail.com>
Date: Thu, 24 Jul 2014 18:18:29 +0530
Subject: [PATCH] i9082: Add haptic feedback to back and menu softkeys

The default behavior of softkeys on stock ROM is to provide haptic
vibration on touch. This patch restores that behavior.


Change-Id: I3fc7c1e42da8688b54653413dca00fda9b59c171
---
 data/keyboards/Generic.kl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/keyboards/Generic.kl b/data/keyboards/Generic.kl
index 1413319..72ade4f 100644
--- a/data/keyboards/Generic.kl
+++ b/data/keyboards/Generic.kl
@@ -158,7 +158,7 @@ key 128   MEDIA_STOP
 # key 136 "KEY_FIND"
 # key 137 "KEY_CUT"
 # key 138 "KEY_HELP"
-key 139   MENU              WAKE_DROPPED
+key 139   MENU              VIRTUAL
 key 140   CALCULATOR
 # key 141 "KEY_SETUP"
 key 142   POWER             WAKE
@@ -177,7 +177,7 @@ key 152   POWER             WAKE
 key 155   ENVELOPE
 key 156   BOOKMARK
 # key 157 "KEY_COMPUTER"
-key 158   BACK              WAKE_DROPPED
+key 158   BACK              VIRTUAL
 key 159   FORWARD
 key 160   MEDIA_CLOSE
 key 161   MEDIA_EJECT
-- 
1.9.1
